docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -380,6 +380,12 @@ Required when `actions` are used to send notifications.
 
 * `field_names`: String[] , required
 
+|related_integrations |Object[] a| Fleet integrations the rule depends on. The object has three fields:
+
+* `package` (String, required): Integration package's name EPR uses
```

Comment:
Some users might not know what EPR stands for, or maybe they're unfamiliar with Elastic Package Registry in general. I'm also wondering how they'd find out what a given package's name is; maybe we can add a link to help them find the integration/packages they're trying to add? 

I tried looking for a list in [elastic/package-registry](https://github.com/elastic/package-registry) but didn't find anything. Maybe [elastic/integrations/tree/main/packages](https://github.com/elastic/integrations/tree/main/packages)?

Reply:
I don't think we have package name documented somewhere. For example [this](https://docs.elastic.co/integrations/elastic_package_registry) page shows integrations list. Package's name as well as integration's name are in fact identifies and hidden from users. UI fetches integrations list from EPR (of course not directly but via Fleet) and uses that ids for API requests. To be 100% sure about id correctness users should use EPR. EPR allows to search for a package while the package contains integrations. After that package or integration ids can be saved since these a stable.

@joepeeples what do you think the best approach to describe it?

Reply:
@maximpn Simpler is probably better; we can link users to the EPR repo, which documents the API requests that can help them search for the right packages and integrations.
```suggestion
* `package` (String, required): The integration package's name, as used by the https://github.com/elastic/package-registry[Elastic Package Registry].
```